devicetypes/sky-nie/min-smart-plug.src/min-smart-plug.groovy

```diff
@@ -49,28 +68,62 @@ metadata {
         capability "Configuration"
         capability "Refresh"
         capability "Health Check"
+        capability "Button"
 
         attribute "firmwareVersion", "string"
         attribute "syncStatus", "string"
 
         fingerprint mfr: "0312", prod: "C000", model: "C009", deviceJoinName: "Minoston Outlet" // old MP21Z
         fingerprint mfr: "0312", prod: "FF00", model: "FF0C", deviceJoinName: "Minoston Outlet" //MP21Z Minoston Mini Smart Plug
         fingerprint mfr: "0312", prod: "AC01", model: "4001", deviceJoinName: "New One Outlet"  // N4001 New One  Mini Smart Plug
+        fingerprint mfr: "0312", prod: "EE00", model: "EE01", deviceJoinName: "Minoston Switch" //MS10ZS Minoston Smart Switch
+        fingerprint mfr: "0312", prod: "EE00", model: "EE03", deviceJoinName: "Minoston Switch" //MS12ZS Minoston Smart on/off Toggle Switch
+        fingerprint mfr: "0312", prod: "A000", model: "A005", deviceJoinName: "Evalogik Switch" //ZW30
+        fingerprint mfr: "0312", prod: "BB00", model: "BB01", deviceJoinName: "Evalogik Switch" //ZW30S Evalogik Smart on/off Switch
+        fingerprint mfr: "0312", prod: "BB00", model: "BB03", deviceJoinName: "Evalogik Switch" //ZW30TS Evalogik Smart on/off Toggle Switch
```

Comment:
if these are switches you can change the icon they use in the app to a switch by adding `ocfDeviceType: "oic.d.switch"` to the end of these fingerprint lines 

otherwise you'll get the smart plug icon

Reply:
Yes, you are right! I modified them.